controllers/scopetemplate_controller.go

```diff
@@ -50,7 +50,8 @@ type ScopeTemplateReconciler struct {
 
 const (
 	// generateNames are used to track each binding we create for a single scopeTemplate
-	clusterRoleGenerateKey = "operators.coreos.io/generateName"
+	clusterRoleGenerateKey            = "operators.coreos.io/generateName"
+	scopeTemplateControllerFieldOwner = "scopetemplate-controller"
```

Comment:
It's really long variable name :) and since these constants are not exported, one can assume they're for the scopeTemplateController

```suggestion
	controllerFieldOwner = "scopetemplate-controller"
```

Reply:
Makes sense to shorten it. I also wonder if it would be best to have only one field owner variable with a value of something like `oria-operator` instead of one for each individual controller?

Reply:
should be resolved in commit https://github.com/awgreene/oria-operator/pull/24/commits/adc34c08f252eb9c231f409898d6c84b07678d98

Reply:
one thing to note, since they are both in the same Go package the names had to be somewhat distinct so they are prefixed with the initials of the CRD the controller is for.